src/pages/components/dropdown/usage.mdx

```diff
@@ -152,111 +167,449 @@ Another important difference between the two components is the underlying code.
 
-## Sample data
-
-```javascript
-const items = [
-  {
-    id: "option-1",
-    text: "Option 1",
-  },
-  {
-    id: "option-2",
-    text: "Option 2",
-  },
-  {
-    id: "option-3",
-    text: "Option 3",
-  },
-  {
-    id: "option-4",
-    text: "Option 4",
-  },
-];
-```
-
-## Variations
-
-| Type                          | Purpose                                                                                                                                                                                 |
-| ----------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| _Dropdown_                    | User needs to pick one option from a list.                                                                                                                                              |
-| _Filter dropdown_             | When a list contains more than 25 items, use filtering to help find options from the list. May only be used with dropdown or multiselect dropdown. Cannot be used with inline dropdown. |
-| _Multiselect dropdown_        | User needs to select multiple options from a dropdown.                                                                                                                                  |
-| _Inline dropdown_             | A stylized dropdown that can be formatted to appear inline with other content and allows the user to select one option.                                                                 |
-| _Inline multiselect dropdown_ | A stylized dropdown that can be formatted to appear inline with other content and allows the user to select more than one option.                                                       |
+## Formatting
+
+### Anatomy
+
+Dropdowns are composed of four distinct sections—assistive text like labels or help text, a field, a menu, and options contained within the menu. Labels and help text are optional but can guide the user to make an informed decision when making a selection.
+
+<Row>
+<Column colLg={8}>
+
+![Dropdown anatomy](images/dropdown-usage-1.png)
+
+</Column>
+</Row>
+
+1. **Labels and help text (optional):** Assistive text to help the user make a selection.
+2. **Field:** Persists when the dropdown is open or closed.
+3. **Option:** A list of multiple options to select from within a menu.
+4. **Menu:** Displayed as an open state and contains a list of options to select from.
+
+### Sizing
+
+#### Height
+
+There are three dropdown height sizes: Large, medium (default) and small (In code these sizes are currently called Extra large, default, and small.). Use a consistent dropdown height when it is being used alongside other form components on the same page. Supporting three different dropdown sizes gives you more flexibility when structuring layouts. When in doubt, use the default medium size height.
```

Comment:
I bet we could have the note around the mismatch in code over on the code page if that'd help to clear this section up. Also can make an issue if that'd be helpful to rectify this.

Reply:
yes pleaseeee 🙏 . That would be helpful.